packages/scripts/README.md

```diff
@@ -1,3 +1,12 @@
 # Glint scripts
 
-A handful of tools for working with Glint—e.g. for migrations.
\ No newline at end of file
+A handful of tools for working with Glint—e.g. for migrations.
+
+## `migrate-glintrc`
+
+To migrate from `.glintrc.yml` files to using a `glint` key within `tsconfig.json` files:
+
+- install `@glint/scripts` as a dev dependency
+- run the `migrate-glintrc` script:
+    - `node ./node_modules/.bin/migrate-glintrc <paths to glintrc.yml files to migrate>`
+    - `yarn migrate-glintrc <paths to glintrc.yml files to migrate>`
```

Comment:
An alternative that avoids needing to explicitly install the package and deal with `.bin` stuff might be:

```
npx -p @glint/scripts migrate-glintrc <path(s) to glintrc.yml files to migrate>
```

Reply:
Ooooh, today I learned. I went looking for that thing and could not find it anywhere. Search fail!